nmdc_automation/config/workflows/workflows.yaml

```diff
@@ -285,52 +285,47 @@ Workflows:
 
 
   - Name: MAGs
-    Type: nmdc:MagsAnalysisActivity
+    Type: nmdc:MagsAnalysis
     Enabled: True
     Analyte Category: Metagenome
     Git_repo: https://github.com/microbiomedata/metaMAGs
-    Version: v1.3.10
+    Version: v1.3.4
```

Comment:
same here, merge conflict did not pick the right version, this should be v1.3.10

Reply:
updated